patches/patch1.diff

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/org.gnome.Characters.in b/src/org.gnome.Characters.in
```

Comment:
Please add a comment at the top of this patch saying what it is for.

Reply:
Done.